### DISCUSSION TITLE
Builder c++ packages

### DISCUSSION THREAD

**Discussion by CyberGonzo** [DelphiPraxis/The-Drag-and-Drop-Component-Suite-for-Delphi#54] [Q&A]
Hi, will you consider creating C++ Builder packages ?

**Comment by landrix** [comment]
i can't give you any C++ suupport.

**Reply by CyberGonzo** [reply]
FYI, I created a static library project and built lib/a files from the *.pas files.  That works great.

One thing that bugs me is that the 64 bit **Release** built *.a file is 'huge' compared to the 32 bit *.lib file.

.lib = 391 KB
.a   = 2147 KB  (more than 5 times bigger than the .lib file)

My question, is this true for your component built library files as well ?
I have been going through the Delphi compiler settings but cannot find anything that may explain this.
As said, it's built in **release** mode, so no debug information is linked in.

I did disable Delphi Compiler\Assertions and that knocked off 4 KB of the libraries (387 and 2143 KB) but the huge *.a file bugs me.

PS. I use C++ Builder 11
